kythe/cxx/common/path_utils.cc

```diff
@@ -314,6 +314,9 @@ std::string JoinPath(absl::string_view a, absl::string_view b) {
 
 bool AbslParseFlag(absl::string_view text, PathCanonicalizer::PathEntry* entry,
                    std::string* error) {
+  if (text.empty()) {
+    return true;
```

Comment:
When returning `true` the `entry` needs to be set to something, likely just `*entry = {}`.

The `absl::StrSplit` in `vector<PathCanonicalizer::PathEntry>` below probably needs to `SkipEmpty` as a more comprehensive fix.

Reply:
:+1: changed to use `SkipEmpty` below and removed the `text.empty()` check.

Do we still need something like `*entries = {}` or will `entries` point to a default-constructed type?

Reply:
I think omitting that is fine.  Abseil flags should support `std::optional<T>` flags these days to provide a reasonable default.